src/components/LanguageListItemCard/LanguageListItemCard.tsx

```diff
@@ -10,41 +16,39 @@ import { Sensitivity } from '../Sensitivity';
 import { TogglePinButton } from '../TogglePinButton';
 import { LanguageListItemFragment } from './LanguageListItem.graphql';
 
-const useStyles = makeStyles()(({ spacing }) => {
-  return {
-    root: {
-      width: '100%',
-      maxWidth: 400,
-      position: 'relative',
-    },
-    name: {
-      marginBottom: spacing(2),
-    },
-    bottomSection: {
-      display: 'flex',
-      justifyContent: 'space-between',
-      alignItems: 'flex-end',
-    },
-    leftContent: {
-      flex: 1,
-    },
-    rightContent: {
-      marginLeft: spacing(2),
-      display: 'flex',
-      flexDirection: 'column',
-      justifyContent: 'flex-end',
-      textAlign: 'right',
-    },
-    sensitivity: {
-      marginTop: spacing(1),
-    },
-    pin: {
-      position: 'absolute',
-      top: 10,
-      right: 10,
-    },
-  };
-});
+const classes = {
```

Comment:
Hmm I'm not sure I like this. Downsides I can first think of is not having type completion on these css props and if there's an error it would be shown at the usage instead of this declaration. What do you think about just inlining these objects?

Reply:
Inlining them makes sense. This first commit was mainly to see what direction we want to go with the and I agree it's nicer to get and error on declaration rather than usage